doc/lua_api.md

```diff
@@ -3261,10 +3261,12 @@ Elements
       Exception: defaults to 0 for indent columns
     * `tooltip=<value>`: tooltip text (default: empty)
     * `image` column options:
+        * `vpadding=<value>`: vertical padding in pixels (default: `2`).
+          applied at both the top and bottom of the cell.
```

Comment:
Expected question: Other properties use em. Why pixels instead of em?

Answer: I think it wouldn't make sense for the vertical padding of images to depend on font size. If the font size and thus the row height increases, I want the padding to stay the same and the image to use the extra space.